repository_files.go

```diff
@@ -82,6 +86,54 @@ func (s *RepositoryFilesService) GetFile(pid interface{}, fileName string, opt *
 	return f, resp, err
 }
 
+
+
+
+
+// GetFileOptions represents the available GetFile() options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/repository_files.html#get-file-from-repository
+type GetFileMetaOptions struct {
+	Ref *string `url:"ref,omitempty" json:"ref,omitempty"`
+}
+
+// GetFile allows you to receive information about a file in repository like
```

Comment:
This should be `GetFileMeta`, but maybe we should call them (this one and the options struct `GetFileMetaData`? Sounds better to me...

Reply:
changed